node-react/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "root",
+  "version": "0.0.0",
```

Comment:
Probably should make this `1.0.0` just to look like we did something 😄

Reply:
:)